backend/wmg/data/wmg_cube.py

```diff
@@ -36,3 +48,216 @@ def create_cell_count_cube(tdb_group: str):
         )
 
         tiledb.from_pandas(uri, df)
+
+
+def create_cubes(tdb_group):
+    create_expression_summary_cube(tdb_group)
+    create_cell_count_cube(tdb_group)
+
+
+def create_expression_summary_cube(tdb_group):
+    """
+    Create queryable cube and write to disk
+    """
+    uri = f"{tdb_group}/cube"
```

Comment:
not the name ultimately used by the API layer.  is it mapped later to correct name later on? if so, probably better to use the expected name throughout; we have a constant for it: backend/wmg/data/snapshot.py:17

Reply:
updated with the correct constant